src/track/beats.h

```diff
@@ -27,6 +27,17 @@ using BeatList = QList<track::io::Beat>;
 class Track;
 
 namespace mixxx {
+/// This is an intermediate class which encapsulates the beats into a
+/// plain copyable, movable object.
+class BeatsInternal {
+  private:
```

Comment:
I thought the intention was to make this usable in realtime code without the QObject, so shouldn't these be public? @daschuer could you take a look?

Reply:
There is nothing against using a QObject in real time code.
You just can't receive signals and you should not copy it around.
I am in doubt that there is a need for copying it because.